### DISCUSSION TITLE
Say ‘Hi Lexin’ to Start! does nothing.

### DISCUSSION THREAD

**Discussion by DustyMustard** [toverainc/willow#181] [Q&A] [answered]
Neither my girlfriend nor I saying ‘Hi Lexin’ does anything at all that I can tell.

What should I look into for troubleshooting this? It’s a little beyond me. Have reflashed several times with different changes to settings but no go.

Is a correct connection to HA required for Willow to visually do something when I say ‘Hi Lexin’?

Also, anyone know of a YouTube channel on configuring these? Otherwise once I figure this thing out I’ll start recording videos of the setup process for people that don’t have proper experience with things like this such as myself (my partner is a software engineer so I can ask her questions when needed).

**Comment by nikito** [comment, marked as answer]
I think the problem is that "Lexin" in this case is Chinese, so it isn't pronounced "Lex-in", I think it is something like "Lua-shin". I've used the "Hi ESP" phase with much better results 😂

**Reply by kristiankielhofner** [reply]
This is exactly right. We likely won't include "Hi Lexin" in Willow 1.0 because of this issue.

**Comment by blakadder** [comment]
It does work https://twitter.com/blakadder_/status/1670179382634196993?s=20 but its practicality is questionable for non chinese speakers.

This is how it's supposed to sound 

https://github.com/toverainc/willow/assets/5904370/547bd99f-13d5-4007-90d0-74d6ff960895
